### ISSUE TITLE
How to handle trailing slashes

### ISSUE THREAD

**Issue by jsonmaur** [reach/router#101]
How would one handle forcing trailing slashes with the reach router? I tried using `Redirect` manually since I couldn't find any global props for it, but as the path ranking seems to ignore trailing slashes, this results in an infinite redirect loop:

```jsx
<Router>
  <Redirect from="/path" to="/path/" />
  <Component path="/path/" />
</Router>
```

**Comment by doronnac** [comment]
+1
from my understanding, a trailing slash is helpful in describing a "folder", or a "base uri" in a server-side router, in accordance to [rfc 3986](https://tools.ietf.org/html/rfc3986), which describes a standard path resolution including relative paths (check section 5).
I believe it's also similar to how a unix path resolution works.

imo it might be worthwhile to use this as a convention.

**Comment by unframework** [comment]
I am also interested in this answer: would be great to have a "strict" mode for matching trailing slashes. Another way to look at a trailing slash is as creating a zero-length path segment at the end - would be great to see a way to handle this more explicitly.

**Comment by bartekus** [comment]
Yes, this is quite important feature, and I'd be happy to submit a PR with a solution to this problem.
Perhaps as suggested, having 'strict' flag on the Redirect component is the best approach here!?

**Comment by multidis** [comment]
I wonder if there is a way to enforce a redirect to have consistent URLs from an SEO perspective. For example, a component

`<Component path="app/someitem/:itemId" />`

Becomes available at both `/app/someitem/12345` and `/app/someitem/12345/`
